github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -128,6 +128,38 @@ presubmits:
           resources:
             requests:
               memory: "29Gi"
+  - name: pull-kubevirt-e2e-k8s-1.16.2
+    skip_branches:
+      - release-\d+\.\d+
+    annotations:
+      fork-per-release: "true"
+    always_run: true
```

Comment:
You could also set `always_run` to `false`. Then we can merge this job right away.